bin/controller.ts

```diff
@@ -28,9 +28,19 @@ const job1: Job = new NormalJob('first', new Set([job4, job2]))
 const job5: Job = new NormalJob('fifth', new Set([job1]))
 
 const client: Client = new Http2Client(new HeapJobOrderer([job5]))
-client.on('progress', console.log)
+client.on('progress', (FinishedJob) => {
```

Comment:
Why is the variable `FinishedJob` uppercase?

Reply:
My mistake. Will change.